content/index.rst

```diff
@@ -69,7 +69,7 @@ for recommended prerequisite knowledge.
    30 min ; :doc:`non-blocking-communication-pt2`
    30 min ; :doc:`one-sided-pt1`
```

Comment:
@robertodr What do you plan here and at line 63? Please push a change that suits you!

Reply:
You mean time-wise?

Reply:
Yes, so we have input to fix up https://enccs.se/events/2020/intermediate-topics-in-mpi/

Reply:
30 minutes each for comms&groups and datatypes. And hopefully 60 and 45 minutes for one-sided part 1 and one-sided part 2, respectively, will be enough.